code/game/objects/structures/grille.dm

```diff
@@ -84,6 +84,7 @@
 		return
 	var/mob/M = AM
 	shock(M, 70)
+	take_damage(rand(0,1), BRUTE, "melee", FALSE)
```

Comment:
what's the point of calling damage with 0?
```suggestion
	if(rand(0,1))
		take_damage(1, BRUTE, "melee", FALSE)
```

Reply:
🤷